magellan-sample-advanced/src/main/java/com/wealthfront/magellan/sample/advanced/MainMenuStep.kt

```diff
@@ -74,6 +74,18 @@ class MainMenuStep(
     binding.bottomBarNavigation.selectedItemId = selectedTab
   }
 
+  override fun onBackPressed(): Boolean {
+    if (navigator.backPressed()) {
```

Comment:
TODO don't need this. I believe LifecycleComponent's backPressed already does what we want